docs/docs/draft/documentation/README.md

```diff
@@ -1312,12 +1312,76 @@ curl -XDELETE http://localhost:5681/meshes/mesh-1/traffic-logs/tl-1
 The [`kumactl`](/kumactl) CLI under the hood makes HTTP requests to this API.
 :::
 
+## Security
+
+Kuma helps you secure your current infrastructure with mTLS. The following sections cover details of how it works.
+
+### Certificates
+
+Kuma uses a built-in CA (Certificate Authority) to issue certificates for dataplanes. The root certificate is unique for each mesh
+in the system. On Kubernetes, certificates are stored as a [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/).
```

Comment:
From this sentence I would imply that all certificates (including ones for dataplanes) get stored.